app/styles/network/show/details.scss

```diff
@@ -22,9 +24,10 @@ div#details-sidebar {
 
   /*change network name*/
   div.network-input {
+    margin-bottom: 0px;
     margin-top: 20px;
-    input { height:  50px !important;}
-    .controls { line-height: 60px;}
+    input { height:  35px !important; margin-bottom: 0px}
```

Comment:
Leave an empty space before closing brace `}`.
And you forgot the semicolon at the end of the line.